app/services/books/book_rating.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Books
+  class BookRating < ::Callable
+    def initialize(book:)
+      @book = book
+    end
+
+    def call
+      votes = {}
+      table = Rating
+              .select('ratings.title, count(ratings)')
+              .joins('LEFT JOIN rating_books ON ratings.id = rating_books.rating_id')
+              .where('rating_books.book_id = ?', book)
```

Comment:
you can use [hash conditions](https://guides.rubyonrails.org/active_record_querying.html#hash-conditions) here. As for me they are more readable in this case